src/cascadia/TerminalControl/TerminalControl.vcxproj.filters

```diff
@@ -11,20 +11,15 @@
   </ItemGroup>
   <ItemGroup>
     <ClCompile Include="pch.cpp" />
-    <ClCompile Include="KeyChord.cpp" />

```

Comment:
What's going on here? Why are these getting removed? (and below)

It doesn't look like you otherwise manipulated this module. Were these moved out by someone else and the filters wasn't updated?

Reply:
I have no idea! I guess so, as the .vcxproj wasn't changed.